wiki/Article_Style_Guide/News/en.md

```diff
@@ -46,7 +46,7 @@ News articles are markdown files (``.md``) are named under the following format:
 
 Where ``yyyy`` is year as 4 digits, ``mm`` is month as 2 digits, and ``dd`` is the date as 2 digits.
 
-Article title strings for file names are the title of the article itself with all spaces replaced with hyphens (`-`), and all additional characters such as periods (`.`), exclamation marks (`!`), question marks (`?`) and commas (`'`) removed with no replacement.
+Article title strings for file names are the title of the article itself with all spaces replaced with hyphens (`-`), and all additional characters such as periods (`.`), exclamation marks (`!`), question marks (`?`) and commas (`,`) removed with no replacement.
```

Comment:
I would assume certain invalid filename characters would also fall into this list too 🤔 (that is, `>`, `<`, `:`, `"`, `/`, `\`, `|`, and `*` ... you already covered the `?`)